glui_control.cpp

```diff
@@ -313,12 +319,18 @@ int GLUI_Control::char_width(char c)
 { /* Hash table for faster character width lookups - JVK
        Speeds up the textbox a little bit.
   */
-  int hash_index = c % CHAR_WIDTH_HASH_SIZE;
+  /*Killing this with fire.
```

Comment:
I took a second look at this caching scheme for `glutBitmapWidth`.  I think the existing implementation here is more robust than the replacement.  What's the rationale for the change?  I'm a bit tempted to turn the 2D array into a 1D array of structs for clarity.  Perhaps also take the CHAR_WIDTH_HASH_SIZE constant out of the header.  But I think it's moot if/when we can switch to [libdrawtext](http://nuclear.mutantstargoat.com/sw/libdrawtext/).